src/types.ts

```diff
@@ -148,6 +163,8 @@ export interface ItemProps<NodeType extends HTMLElement> extends InternalItem {
   children: (props: ChildrenFnProps<NodeType>) => JSX.Element
 }
 
+export interface DataSourceItem extends InternalItem {}
```

Comment:
mb we also should add export of smth like this

`export type DataSource = DataSourceItem[]`